test/e2e/framework/gobindata_util.go

```diff
@@ -16,22 +16,16 @@ limitations under the License.
 
 package framework
 
+//go:generate ../../../hack/update-bindata.sh
+//go:generate pwd
```

Comment:
i can remove this i guess.  i think its useful but not necessary now that its working smoothly